src/pybroker/data.py

```diff
@@ -503,7 +505,15 @@ def _fetch_data(
         _adjust: Optional[str],
     ) -> pd.DataFrame:
         """:meta private:"""
-        df = yfinance.download(list(symbols), start=start_date, end=end_date)
+        yf_progress_bar_flag = (
+            self._logger.disable_flag and self._logger.progress_bar_flag
```

Comment:
Should this be an or condition?

Reply:
Nevermind, the and is correct. But can you change the property names below and negate?

Reply:
👌，i will fix it